core/FileIntegrity.php

```diff
@@ -332,9 +332,9 @@ protected static function isFileNotInManifestButExpectedAnyway($file)
     protected static function getMessagesFilesMismatch($messages)
     {
         $messagesMismatch = array();
-        $hasMd5file = function_exists('md5_file');
+        $hasHashFile = function_exists('hash_file');
```

Comment:
The `RecommendedFunctionsCheck` is testing for `md5_file`, should we add `hash_file` to the list in `getRecommendedFunctions` to match the other one?

Reply:
yes, makes sense.

Reply:
Now we only need two fresh screenshots for the affected UI tests :eyes: